SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -469,7 +469,7 @@ - (void)ensureThumbnail
     }
 
     if (result == nil) {
-        OWSFail(@"%@ Unable to build thumbnail for attachmentId: %@", self.logTag, self.uniqueId);
+        DDLogError(@"%@ Unable to build thumbnail for attachmentId: %@", self.logTag, self.uniqueId);
```

Comment:
What's this change about? I haven't yet seen this happen except in the case of bad data left over from a bug, and would prefer to keep it.

Reply:
Reverted. I didn't mean to include this.